tools/ci/ethereum_test

```diff
@@ -27,8 +27,6 @@ cd $SRCROOT/integration
 deploy_v05_contracts
 ./service_agreement_test
 
-deploy_v06_contracts
-./flux_monitor_test
 
```

Comment:
If we remove this bash test we should probably make the `geth-postgres-ts` required for merging. I think @se3000 may have to do that? Also be aware that removing this affects both `geth-postgres` and `parity-postgres` and we currently don't have a `parity-postgres-ts` substitute.

Reply:
Ryan is setting up the Parity environment in another PR.